Changes in module 'nf-core/samtools/sort'
--- modules/nf-core/samtools/sort/main.nf
+++ modules/nf-core/samtools/sort/main.nf
@@ -23,7 +23,7 @@
     def prefix = task.ext.prefix ?: "${meta.id}"
     if ("$bam" == "${prefix}.bam") error "Input and output names are the same, use \"task.ext.prefix\" to disambiguate!"
     """
-    samtools sort $args -@ $task.cpus -o ${prefix}.bam -T $prefix $bam
+    samtools sort $args -@ $task.cpus -o ${prefix}.sorted.bam -T $prefix $bam
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//')
@@ -33,7 +33,7 @@
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.bam
+    touch ${prefix}.sorted.bam
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":

************************************************************
